dsp-repository/v1/dsp-repository.shacl.ttl

```diff
@@ -26,32 +26,670 @@
+    rdfs:subPropertyOf schema:distribution .
+
+# http://ns.dasch.swiss/repository#hasGivenName
+dsp-repo:hasGivenName
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasFamilyName
+dsp-repo:hasFamilyName
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasEmail
+dsp-repo:hasEmail
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasAddress
+dsp-repo:hasAddress
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#isMemberOf
+dsp-repo:isMemberOf
+    a rdf:Property ;
+    rdfs:subPropertyOf schema:memberOf .
+
+# http://ns.dasch.swiss/repository#hasJobTitle
+dsp-repo:hasJobTitle
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasRole
+dsp-repo:hasRole
+    a rdf:Property ;
+    rdfs:subPropertyOf prov:hadRole .
+
+# http://ns.dasch.swiss/repository#hasNumber
+dsp-repo:hasNumber
+    a rdf:Property .
+
+# # http://ns.dasch.swiss/repository#hasBooBoo
+# dsp-repo:hasBooBoo
+#     a rdf:Property .
+
+##### CLASS DEFINITIONS #####
+### --- Class definition of dsp-repo:Project --- 
+# http://ns.dasch.swiss/repository#Project
+dsp-repo:Project 
+    a rdfs:Class ;
+    rdfs:label "dsp-repo:Project"@en ;
+    rdfs:comment "A project"@en ;
+    rdfs:subClassOf schema:Project , prov:Entity .   
+
+### --- Class definition of dsp-repo:Dataset --- 
+# http://ns.dasch.swiss/repository#Dataset
+dsp-repo:Dataset 
+    a rdfs:Class ;
+    rdfs:comment "A dataset"@en ;
+    rdfs:label "dsp-repo:Dataset"@en ;
+    rdfs:subClassOf schema:Dataset .
+
+### --- Class definition of dsp-repo:Person --- 
+# http://ns.dasch.swiss/repository#Person
+dsp-repo:Person 
+    a rdfs:Class ;
+    rdfs:comment "A person"@en ;
+    rdfs:label "dsp-repo:Person"@en ;
+    rdfs:subClassOf schema:Person , prov:Agent . 
+
+### --- Class definition of dsp-repo:Organization --- 
+# http://ns.dasch.swiss/repository#Organization
+dsp-repo:Organization 
+    a rdfs:Class ;
+    rdfs:comment "An organization"@en ;
+    rdfs:label "dsp-repo:Organization"@en ;
+    rdfs:subClassOf schema:Organization , prov:Agent . 
+
+### --- Class definition of dsp-repo:Grant --- 
+# http://ns.dasch.swiss/repository#Grant
+dsp-repo:Grant 
+    a rdfs:Class ;
+    rdfs:comment "A grant"@en ;
+    rdfs:label "dsp-repo:Grant"@en ;
+    rdfs:subClassOf schema:Grant . 
+
+### --- Class definition of dsp-repo:DataManagementPlan --- 
 # http://ns.dasch.swiss/repository#DataManagementPlan
 dsp-repo:DataManagementPlan
-    a owl:Class, sh:NodeShape ;
-    rdfs:label "dsp-repo:DataManagementPlan"@en ;
+    a rdfs:Class ;
     rdfs:comment "A Data Management Plan"@en ;
-    rdfs:subClassOf schema:Thing ;
+    rdfs:label "dsp-repo:DataManagementPlan"@en ;
+    rdfs:subClassOf schema:Thing .
+
+##### SHAPES #####
+### --- Shape of dsp-repo:Project ---
+dsp-repo:ProjectShape
+    a sh:NodeShape ;
+    sh:targetClass dsp-repo:Project ;
+    sh:property [
+        sh:path dsp-repo:hasName ;
+        sh:name "Name" ;
+        sh:description "Name of the project"@en ;
```

Comment:
Please see https://www.w3.org/TR/shacl/#name: 
2.3.2.1 sh:name and sh:description
Property shapes may have one or more values for sh:name to provide human-readable labels for the property in the target where it appears. If present, tools should prefer those locally specified labels over globally specified labels at the rdf:Property itself. For example, if a form displays a node that is in the target of a given property shape with an sh:name, then the tool should use the provided name. Similarly, property shape may have values for sh:description to provide descriptions of the property in the given context. Both sh:name and sh:description may have multiple values, but should only have one value per language tag.

Reply:
Yes I read this already
> Both sh:name and sh:description may have multiple values, but should only have one value per language tag.

Thence my comment, we have
```
sh:name "Name" ;
sh:description "Name of the project"@en ;
```

I understand we can't have 

```
sh:name "Name" ;
sh:description "Name of the project"@en , "Project's name"@en ;
```

But why can't we have : 
```
sh:name "Name"@en ;
sh:description "Name of the project"@en ;
```
? 

I am sorry, I am feeling a bit dummy, here... But I would like to make sure I understand :-)

Reply:
Maybe you can. But now that you mentioned it, I think we should remove all the language tags since we only need one working version for now. And leave the different languages for later for a user story.

Reply:
I agree. But should we remove them all or assign everywhere the `@en` tag?